org.eclipse.winery.bpmn4tosca.converter.tobpel/src/test/java/org/eclipse/winery/bpmn2bpel/BPMN4Tosca2BpelTest.java

```diff
@@ -25,38 +27,239 @@
 import org.junit.Before;
 import org.junit.Test;
 
+import static org.junit.Assert.fail;
 
+/**
+ * BPMN4Tosca2BpelTest Class for Testing.
+ */
 public class BPMN4Tosca2BpelTest {
+    /**
+     * Resources Dir for Test Files.
+     */
+    private static final String RESOURCESDIR =
+            "src/test/resources/bpmn4tosca";
 
-	protected static String RESOURCES_DIR = "src/test/resources/bpmn4tosca";
-
-
-	@Before
-	public void setUp() throws Exception {
-
-	}
-
-	@After
-	public void tearDown() throws Exception {
-	}
-
-	@Test
-	public void testTransform() throws ParseException, PlanWriterException, MalformedURLException, URISyntaxException {
-		URI srcUri = Paths.get(RESOURCES_DIR, "bpmn4tosca.json").toUri();
-		URI targetUri = Paths.get(RESOURCES_DIR, "managementplan.zip").toUri();
-		BPMN4Tosca2BpelTest.class.getResource(".");
-		Bpmn4Tosca2Bpel transformer = new Bpmn4Tosca2Bpel();
-		transformer.transform(srcUri, targetUri);
-	}
-	
-	@Test
-	public void testTransformGateway()
-			throws ParseException, PlanWriterException, MalformedURLException, URISyntaxException {
-		URI srcUri = Paths.get(RESOURCES_DIR, "bpmn4tosca.exclusivegateway.json").toUri();
-		URI targetUri = Paths.get(RESOURCES_DIR, "managementplan.exclusivegateway.zip").toUri();
-		BPMN4Tosca2BpelTest.class.getResource(".");
-		Bpmn4Tosca2Bpel transformer = new Bpmn4Tosca2Bpel();
-		transformer.transform(srcUri, targetUri);
-	}
+    /**
+     * Timestamp for defining actual Test Results.
+     */
+    private String timeStamp;
+
+    /**
+     * Setup everything needed.
+     *
+     * @throws Exception
+     *         If the Pattern is not correct.
+     */
+    @Before
+    public void setUp() throws Exception {
+        timeStamp = new SimpleDateFormat(
+                "yyyy.MM.dd.HH.mm.ss")
+                .format(new Date());
+    }
+
+    /**
+     * Teardown after all Tests.
+     *
+     * @throws Exception
+     *         If something goes wrong.
+     */
+    @After
+    public void tearDown() throws Exception {
+    }
+
+    /**
+     * Tests the Tranform  Json  to Zip Functionality.
+     *
+     * @throws ParseException
+     *         If json contains errors.
+     * @throws PlanWriterException
+     *         if Plan writer has problems.
+     * @throws MalformedURLException
+     *         If file paths are not defined correctly.
+     * @throws URISyntaxException
+     *         If the Syntax of the URI contains an Error.
+     */
+    @Test
+    public void testTransform() throws ParseException,
+            PlanWriterException, MalformedURLException,
+            URISyntaxException {
+        URI srcUri = Paths.get(RESOURCESDIR,
+                "bpmn4tosca.json").toUri();
+        URI targetUri = Paths.get(RESOURCESDIR,
+                timeStamp + "managementplan.zip").toUri();
+        BPMN4Tosca2BpelTest.class.getResource(".");
+        Bpmn4Tosca2Bpel transformer = new Bpmn4Tosca2Bpel();
+        transformer.transform(srcUri, targetUri);
```

Comment:
What about the test result? Why is the test result not compared with something existing?